js/src/components/shipping-rate-section/estimated-shipping-rates-card/estimated-shipping-rates-card.js

```diff
@@ -45,33 +45,6 @@ export default function EstimatedShippingRatesCard( {
 		getDeleteHandler,
 	} = getHandlers( { value, onChange } );
 
-	/**
-	 * An Edit button that displays EditRateFormModal to edit shipping rate group upon clicking on the Edit button.
-	 *
-	 * @param {Object} props Props.
-	 * @param {Array<CountryCode>} props.countryOptions Country options to be passed to EditRateFormModal.
-	 * @param {ShippingRateGroup} props.group Shipping rate group to be edited.
```

Comment:
💅  Its `@typedef` could be removed along with it.

https://github.com/woocommerce/google-listings-and-ads/blob/20498e22cd70dacb077fd235b1e851bb17db5793/js/src/components/shipping-rate-section/estimated-shipping-rates-card/estimated-shipping-rates-card.js#L24